keps/sig-storage/3476-volume-group-snapshot/README.md

```diff
@@ -368,22 +379,12 @@ Type VolumeGroupSnapshotStatus struct {
         CreationTime *metav1.Time
 
         // +optional
-        Error *VolumeGroupSnapshotError
-
-        // List of volume snapshots
-        // +optional
-        SnapshotList []VolumeSnapshot
-}
-
-// Describes an error encountered on the group snapshot
-type VolumeGroupSnapshotError struct {
-        // time is the timestamp when the error was encountered.
-        // +optional
-        Time *metav1.Time
+        Error *VolumeSnapshotError
 
-        // message details the encountered error
+        // List of volume snapshot refs
+        // The max number of snapshots in the group is 100
         // +optional
-        Message *string
+        VolumeSnapshotRefList []core_v1.ObjectReference
```

Comment:
Since these can only be snapshot kinds, we could make our own SnapshotReference type that only includes name.

Reply:
If we only need snapshot names, can it just be a list of strings?

Reply:
We can follow up on this during a more detailed API review, but I believe the guidance is to define a type scoped to your supported sources.

Reply:
Sure